db-service/lib/cqn2sql.js

```diff
@@ -387,7 +387,7 @@ class CQN2SQLRenderer {
    */
   limit({ rows, offset }) {
     if (!rows) throw new Error('Rows parameter is missing in SELECT.limit(rows, offset)')
-    return !offset ? rows.val : `${rows.val} OFFSET ${offset.val}`
+    return !offset ? this.val(rows) : `${this.val(rows)} OFFSET ${this.val(offset)}`
```

Comment:
Does this use placeholders only for LIMIT? Doesn't seem to me this way.

Reply:
The limit is only inlined when using HANA. Here: https://github.com/cap-js/cds-dbs/pull/567/files#diff-11bd20d3012fd985095f5a9be4a1da3e94e09eac409545ac67688c9582d1d201R711